### PR TITLE
Changed fedramp check away from reconciler, anonymized alert details …

### PR DESCRIPTION
**Pull request by luis-falcon**
Changed fedramp check away from reconciler, anonymized alert details for fedramp clusters

### CODE REVIEW

**Review by blrm — CHANGES_REQUESTED** [review]
I made one comment about removing one more field that includes PII but aside from that, this looks great!

**Review by blrm — APPROVED** [review]

File: pkg/controller/secret/secret_controller.go [hunk 1]
```diff
@@ -317,20 +315,30 @@ func createOCMAgentReceiver(ocmAgentURL string) []*alertmanager.Receiver {
...
+		"alert_name":   `{{ .CommonLabels.alertname }}`,
+		"link":         `{{ if .CommonAnnotations.runbook_url }}{{ .CommonAnnotations.runbook_url }}{{ else if .CommonAnnotations.link }}{{ .CommonAnnotations.link }}{{ else }}https://github.com/openshift/ops-sop/tree/master/v4/alerts/{{ .CommonLabels.alertname }}.md{{ end }}`,
+		"ocm_link":     fmt.Sprintf("https://console.redhat.com/openshift/details/%s", clusterID),
+		"num_firing":   `{{ .Alerts.Firing | len }}`,
+		"num_resolved": `{{ .Alerts.Resolved | len }}`,
+		"resolved":     `{{ template "pagerduty.default.instances" .Alerts.Resolved }}`,
+		"cluster_id":   clusterID,
+	}
+
+	if config.IsFedramp() {
```
Comment (fahlmant) [inline pkg/controller/secret/secret_controller.go:328]: Going with my above comment, there could be one template for standard clusters and one template for fedramp clusters.

Comment (fahlmant) [reply pkg/controller/secret/secret_controller.go:328]: No I mean a different file. Then if fedramp, load the fedramp version of the file

**Review by fahlmant — APPROVED** [review]
/approve

**Pull request merged**